### DISCUSSION TITLE
q-cli sample: origin of Git commit email adress?

### DISCUSSION THREAD

**Discussion by mthmulders** [jreleaser/jreleaser#561] [Q&A] [answered]
I finally found some time to dive into JReleaser, and I'm trying to see if I can use it to release a native executable from my project.

I'm looking at the [q-cli sample](https://github.com/aalmiray/q-cli/) and more specifically, the [GitHub "release" action](https://github.com/aalmiray/q-cli/blob/main/.github/workflows/release.yml). It [configures](https://github.com/aalmiray/q-cli/blob/main/.github/workflows/release.yml#L192) an email address for the Git commit(s). I'm curious where that address comes from? Or is it just random?

**Comment by aalmiray** [comment, marked as answer]
I found that particular address in this page https://github.community/t/github-actions-bot-email-address/17204/4
You can of course use any other address you want 😏
